cnxepub/html_parsers.py

```diff
@@ -197,18 +201,34 @@ def keywords(self):
 
     @property
     def license_url(self):
-        items = self.parse('.//xhtml:*[@data-type="license"]/@href')
+        # Three cases for location of metadata stanza:
+        #  1. direct child of current node
+        #  2. direct child of any ancestor
+        #  3. Top of book (occurs when fetching from root)
+        items = self.parse('ancestor-or-self::*/*[@data-type="metadata"]//*'
+                           '[@data-type="license"]/@href'
+                           ' | /xhtml:html/xhtml:body/*'
```

Comment:
This is case 3 in the comment: when asking the document as a whole what the license is (holding the html root node), the license used to be picked up from the grandchild node (html>body>metadata), but I removed the 'any child' logic, so that specific case (which is reasonable and perhaps common - it's in a least on test) needs to be in there.